geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/StartServerCommand.java

```diff
@@ -301,6 +301,13 @@ ResultModel doStartServer(String memberName, Boolean assignBuckets, String bindA
         ConfigurationProperties.HTTP_SERVICE_PORT, httpServicePort);
     StartMemberUtils.setPropertyIfNotNull(gemfireProperties,
         ConfigurationProperties.HTTP_SERVICE_BIND_ADDRESS, httpServiceBindAddress);
+
+    // if redis-port or redis-bind-address are specified in the command line,
+    // REDIS_SERVICE_ENABLED should be set to true
+    if (StringUtils.isNotBlank(redisPassword) || StringUtils.isNotBlank(redisBindAddress)) {
```

Comment:
I see this checking the password and bindAddress setting. But shouldn't it also be checking the redisPort? The comment says port and bind-address. Perhaps any redis option (including password) should enable redis.

Reply:
You are right. What do you think? Maybe it would make sense to enable it if any redis option is specified?

Reply:
Updated it!